src/main/java/gateway/Application.java

```diff
@@ -88,6 +92,11 @@ public static void main(String[] args) {
 	}
 
 	@Bean
+	public Queue abortJobsQueue() {
+		return new Queue(String.format(JobMessageFactory.KAFKA_TOPIC_TEMPLATE, JobMessageFactory.ABORT_JOB_TOPIC_NAME, SPACE));
```

Comment:
consider renaming this topic template to not contain the word kafka, since it's not specific to kafka.